src/pages/signin/SignInPage.tsx

```diff
@@ -124,7 +126,7 @@ function getRenderOptions({
     };
 }
 
-function SignInPageInner({credentials, account, activeClients = [], preferredLocale}: SignInPageInnerProps) {
+function SignInPageInner({credentials, account, activeClients = [], preferredLocale, shouldEnableMaxHeight = true}: SignInPageInnerProps) {
```

Comment:
Thinking about this, I can't see a good reason to introduce `shouldEnableMaxHeight` as a prop as `SignInPage` isn't being used from anywhere else, so we could just set it to false here. Or am I missing something? :smile:

Reply:
@suneox Friendly bump on this question :bow:

Reply:
Did you test for that case? I think it would also want the same fix applied, no?

Reply:
I meant have you tested for the case where the `SignInModal` is used in `Router`? I don't think I agree that it should have a different value for `shouldEnableMaxHeight`.